Dockerfile.template

```diff
@@ -1,5 +1,6 @@
 FROM registry.ci.openshift.org/origin/4.12:artifacts as artifacts
 FROM registry.ci.openshift.org/origin/4.12:machine-config-operator as mcd
+FROM registry.ci.openshift.org/origin/4.12:rhel-coreos-8 as rhcos
```

Comment:
we can't do this unfortunately if we want any kind of reproducibility out side of builds that have access to `ocp` - `rhel-coreos-8` isn't going to be mirrored to `origin` going forward.

Reply:
Ah, snap, we might want to have src version of `os` then (present in the imagestream, but not included in the payload)

Reply:
that would be nice indeed. I think for now, let's just remove the fcos config, and continue with the `os` submodule?

Reply:
lets wait for https://github.com/openshift/release/pull/31973 to merge and I'll update this PR.